scripts/podman-bootstrap.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e
+
```

Comment:
I think it'd be useful (to me, at least) to have a `$RUNNER` env variable, defaulted to `podman`:

```suggestion
RUNNER=${RUNNER:podman}

```

Then, for example, you can invoke the script like this:

```
RUNNER='sudo podman' ./scripts/podman-bootstrap.sh
```

How have you been running it? With `sudo ./scripts/podman-bootstrap.sh` ?

Reply:
yeh exactly, as this version needs to manipulate networks and at the moment `podman` requires `sudo` privileges for it. not sure I want to keep it that way, but one of the things we will be dropping if going for rootless podman containers is `dns domain name resolution`. the alternative in rootless podman would be to keep all services inside the same pod, and listen to localhost:{service-port} when communicating with other containers.